pkg/detectors/falsepositives.go

```diff
@@ -110,15 +111,43 @@ func FilterResultsWithEntropy(results []Result, entropy float64) []Result {
 	var filteredResults []Result
 	for _, result := range results {
 		if !result.Verified {
-			if result.RawV2 != nil {
-				if StringShannonEntropy(string(result.RawV2)) >= entropy {
+			if result.Raw != nil {
+				if StringShannonEntropy(string(result.Raw)) >= entropy {
 					filteredResults = append(filteredResults, result)
 				}
 			} else {
-				if StringShannonEntropy(string(result.Raw)) >= entropy {
+				filteredResults = append(filteredResults, result)
+			}
+		} else {
+			filteredResults = append(filteredResults, result)
+		}
+	}
+	return filteredResults
+}
+
+// FilterKnownFalsePositives filters out known false positives from the results.
+func FilterKnownFalsePositives(results []Result, falsePositives []FalsePositive, wordCheck bool) []Result {
+	var filteredResults []Result
+	for _, result := range results {
+		if !result.Verified {
+			switch result.DetectorType {
+			case detectorspb.DetectorType_CustomRegex:
+				filteredResults = append(filteredResults, result)
+				break
+			case detectorspb.DetectorType_GCP:
+				filteredResults = append(filteredResults, result)
+				break
+			default:
+				if result.Raw != nil {
+					if !IsKnownFalsePositive(string(result.Raw), falsePositives, wordCheck) {
+						filteredResults = append(filteredResults, result)
+					}
```

Comment:
Do you think it would be useful to trace log that something was filtered? There have been a few times — most recently #2620 — where figuring out why something wasn't returned required adding custom logging.

Reply:
I've extended the results flag with an additional `filtered_unverified` option. It just logs that it was omitted, it doesn't actually 'notify' on it though.